app/javascript/src/components/dashboard/Hours.jsx

```diff
@@ -0,0 +1,242 @@
+import React, {Component} from "react";
+// Semantic Ui Components Used
+import { Dropdown, Card, Grid, Button, Header, Icon, Label, Feed } from "semantic-ui-react";
+//Endpoints not yet created, so Created Fake Data Instead
+import {getHours} from "./mockData";
+
+export default class Hours extends Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      rank: 'hour',
+      hours: getHours(),
+      total: false //Total used to identify if User choose Week or Total View
+    };
+  }
+
+  //Function created to Switch between Total and Week View script for both Hour and Night rank
+
+  hourView() {
+    return this.state.total == false
+      ? "Hours Spent: "
+      : "Hours Spent: ";
+  };
+
+  nightView() {
+    const {total} = this.state;
+    return this.state.total == false
+      ? "Nights Slept: "
+      : "Nights Slept: ";
+  };
+
+  //Function determining which View to Display
+  hoursNumber(user) {
+    return this.state.total == false
+      ? user.hours.week
+      : user.hours.total;
+  };
+
+  nightNumber(user) {
```

Comment:
same thing, arrow functions

Reply:
Take the time to find the rest of your functions that are not arrow functions. I see at least two others that I didn't point out.